### PR TITLE
fix(Bash/Windows Install): Correct path to

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  Currently running the setup on Bash Dashboard on Windows gives this message

```
$ ./acore.sh install-deps
NOTICE: file </d/azerothcore-wotlk/conf/config.sh> not found, we use default configuration only.
Deno version check:
Downloading https://github.com/denoland/deno/releases/download/v1.26.0/deno-x86_64-pc-windows-msvc.zip
Deno is upgrading to version 1.26.0
Upgraded successfully
Platform: msys
/d/azerothcore-wotlk/apps/installer/includes/os_configs/windows.sh: line 3: @%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe: No such file or directory
/d/azerothcore-wotlk/apps/installer/includes/os_configs/windows.sh: line 17: choco: command not found
nothing to do
/d/azerothcore-wotlk/apps/installer/includes/os_configs/windows.sh: line 22: choco: command not found
/d/azerothcore-wotlk/apps/installer/main.sh: line 34: pop_var_context: head of shell_variables not a function context
/d/azerothcore-wotlk/apps/installer/main.sh: line 89: pop_var_context: head of shell_variables not a function context
```

where it can't find the Powershell executable due to an @ 
hopefully this fixes it 

## Issues Addressed:

- Closes 

## Tests Performed:

- 
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Kitzunu** [comment]
this is part of it, but choco commands are also screwed up

@locus313 got any idea?

**Comment by locus313** [comment]
> this is part of it, but choco commands are also screwed up
> 
> @locus313 got any idea?

Looks like its related to the issue with powershell executable. Since it could not find powershell executable it could not run the chocolatey installer. 

Have you tested it again after making this change? Does it still give you the error with choco?

**Comment by Kitzunu** [comment]
@locus313 yes, this makes it find PowerShell, but the choco command is the same

File: apps/installer/includes/os_configs/windows.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # install chocolatey before
 
-@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
```
Comment (locus313) [inline apps/installer/includes/os_configs/windows.sh:3]:
```suggestion
@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "[System.Net.ServicePointManager]::SecurityProtocol = 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
```
Can you give this a try?

**Comment by acidmanifesto** [comment]
Status?

**Comment by Kitzunu** [comment]
I'm unable to test it myself atm

**Comment by Maelthyrr** [comment]
Tested but it doesn't work for me. I get the same error.

**Pull request closed without merging**